administrator/language/de-DE/de-DE.ini

```diff
@@ -75,12 +75,12 @@ JDETAILS="Details"
 JDISABLED="Deaktiviert"
 JENABLED="Aktiviert"
 JFALSE="Falsch"
-JFEATURED="Haupteintrag"
-JFEATURED_ASC="Haupteintrag aufsteigend"
-JFEATURED_DESC="Haupteintrag absteigend"
-JUNFEATURED="Kein Haupteintrag"
-JFEATURE="Haupteintrag"
-JUNFEATURE="Kein Haupteintrag"
+JFEATURED="Hauptbeitrag"
```

Comment:
Ja ich verstehe.
Vielleicht sollte man das in Joomla selbst fixen, indem man separate strings für contact und content hat. Dann kann man das auch bei den Übersetzungen anpassen.